### PR TITLE
Payments ux article

### PR DESCRIPTION
**Pull request by mustafa-x**
Moved article to Payments request directory

### CODE REVIEW

File: src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md [hunk 1]
```diff
@@ -0,0 +1,514 @@
...
+</div>
+
+<div class="clearfix"></div>
+
+
+## What is customisable 
+Some of the UI is customisable with some exceptions such as the 
+colour of button and their respective text labels.
+
+NOTE: This example is for the Chrome browser. Other browsers
```
Comment (PaulKinlan) [inline src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md]: Make this lowercase "note: " - you then get the nice call out in the site.

File: src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md [hunk 2]
```diff
@@ -0,0 +1,514 @@
...
+  </figure>
+</div>
+
+1. The icon is taken from the favicon. Favicon is hi-res and 
+	recognizable for the store. For best practice, provide multiple 
+	icons at different sizes to allow the browser to pick the most 
+	appropriate size. You can [learn more here](/web/fundamentals/discovery-and-monetization/payment-request/deep-dive-into-payment-request). 
+
+
+2. This is taken from the <code>&#60;title&#62;</code> tag.
```
Comment (PaulKinlan) [inline src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md]: I think you can just use back ticks ` to get code highlights.

File: src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md [hunk 3]
```diff
@@ -0,0 +1,514 @@
...
+	        additionalDetailsContainer.focus();
+	    }).catch(function(error) {
+	        // Handle error.
+	    });
+	})
+	.catch(function(error) {
+	    // Handle error.
+	});
+
+You can find code for <a
```
Comment (PaulKinlan) [inline src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md]: Use markdown links

File: src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md [hunk 4]
```diff
@@ -0,0 +1,514 @@
...
+    	can help users tap or click smaller UI elements.  
+	</figcaption>
+  </figure>
+</div>
+
+### User feedback mechanisms
+As well as providing confirmation to users that a 
+purchase was successful, provide feedback to reassure 
+them of the status of their experience. One example of 
+feedback is a <a
```
Comment (PaulKinlan) [inline src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md]: markdown links

File: src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md [hunk 5]
```diff
@@ -0,0 +1,514 @@
...
+the merchant's site without any fuss. The Payment Request 
+API is quick and efficient but existing users will need some 
+education with this new payment flow so focus on introducing 
+the new UX patterns with text labels and UI that express 
+speed and efficiency. 
+
+
+## Code demos and browser support
+You can experiment with the various features and UX patterns 
+with this demo <a href="https://paymentrequest.show/demo/">
```
Comment (PaulKinlan) [inline src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md]: markdown links

File: src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md [hunk 6]
```diff
@@ -0,0 +1,514 @@
...
+paymentrequest.show/demo/</a>. Payments Request API is 
+currently supported by;
+
+* Chrome 53 and above on Android 
+
```
Comment (PaulKinlan) [inline src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md]: remove line

File: src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md [hunk 7]
```diff
@@ -0,0 +1,514 @@
...
+* Chrome 60 and above on Desktop
+
```
Comment (PaulKinlan) [inline src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md]: remove line

File: src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md [hunk 8]
```diff
@@ -0,0 +1,514 @@
...
+* Edge 15 and above on Desktop
+
+You can track the latest status for all other 
+browsers here on <a href="https://caniuse.com/#search=payments">caniuse.com</a>.
```
Comment (PaulKinlan) [inline src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md]: markdown link

File: src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md [hunk 9]
```diff
@@ -0,0 +1,514 @@
...
+
+In this example a user navigates to a site, selects an item and goes
+through the payment request checkout flow. Once a user clicks buy, the
+browser takes over. If the user has previously entered details such as
+their prefered payment method, these will appear as the defaults.
+Otherwise the user can add these. Depending on your particular use
+case you can have defaults auto selected or force the user to fill
+them in.
+
+<video controls autoplay loop muted poster="images/basic-intro-poster.png">
```
Comment (PaulKinlan) [inline src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md]: I think videos have to be in a special `div`

File: src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md [hunk 10]
```diff
@@ -0,0 +1,514 @@
...
+book_path: /web/fundamentals/_book.yaml
+
+{# wf_updated_on: 2017-06-28 #}
+{# wf_published_on: 2017-06-28 #}
+
+
+# Payment Request UX considerations {: .page-title }
+{% include "web/_shared/contributors/mustafa.html" %}
+
+The Payment Request API was fundamentally built for users, helping
```
Comment (PaulKinlan) [inline src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md]: might as well add in a link to the paymentrequest api docs.

File: src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md [hunk 11]
```diff
@@ -0,0 +1,514 @@
...
+them get through burdensome checkout flows on mobile efficiently,
+while providing merchants with a higher conversion rate.
+
+In a study conducted by Google, we found that <strong>65.9% of users
```
Comment (PaulKinlan) [inline src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md]: link to study.

File: src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md [hunk 12]
```diff
@@ -0,0 +1,514 @@
...
+
+Don't block checkout by requiring signup. One of the core benefits of 
+having users already signed in is that you already have payment methods 
+stored for them. Since Payment Request provides easy access to the users 
+payment details, including email and
+phone number, you can simply request these details and avoid requiring
+users to sign in.
+
+### Gracefully handle the UX flow 
+Chrome handles some aspects of the processing, cancellation, and
```
Comment (PaulKinlan) [inline src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md]: new line before this, so consistent with above paragraphs.

File: src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md [hunk 13]
```diff
@@ -0,0 +1,514 @@
...
+</div>
+
+
+<div class="attempt-right">
+  <figure>
+    <img src="images/delivery-options-pickup.png" 
+    alt="This is an example of a label titled 'pickup'">
+  </figure>
+</div>
+Depending on the context the label <em>“Pickup”</em> may be more
```
Comment (PaulKinlan) [inline src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md]: use normal quotes " and not the autoquotes that google docs puts in.

File: src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md [hunk 14]
```diff
@@ -0,0 +1,514 @@
...
+
+
+<div class="attempt-right">
+  <figure>
+    <img src="images/delivery-options-pickup.png" 
+    alt="This is an example of a label titled 'pickup'">
+  </figure>
+</div>
+Depending on the context the label <em>“Pickup”</em> may be more 
+appropriate than the label <em>“Delivery”</em>.
```
Comment (PaulKinlan) [inline src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md]: use normal quotes " and not the autoquotes that google docs puts in.

File: src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md [hunk 15]
```diff
@@ -0,0 +1,514 @@
...
+
+### Midway through the Payments Request 
+You can offer extra UI elements after the payment options 
+have been selected. So instead of showing the API's default 
+spinner, you can show your own, such as in the next example. 
+Then once you have processed the payment details, you can 
+offer up any UI you choose. 
+
+So in this visual example: 
+* The user taps “Buy now” and the merchant calls
```
Comment (PaulKinlan) [inline src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md]: use normal quotes and not auto-quotes.

File: src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md (hunk 11 above)
Comment (mustafa-x) [inline src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md]: it was an internal study

File: src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md (hunk 9 above)
Comment (mustafa-x) [inline src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md]: not according to our articles on the site they done :P 
https://developers.google.com/web/resources/including-videos

File: src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md (hunk 2 above)
Comment (mustafa-x) [inline src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md]: that didnt work here. The title tag was still being added by the browser

File: src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md (hunk 8 above)
Comment (mustafa-x) [inline src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md]: done

File: src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md (hunk 7 above)
Comment (mustafa-x) [inline src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md]: done

File: src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md (hunk 6 above)
Comment (mustafa-x) [inline src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md]: done

File: src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md (hunk 5 above)
Comment (mustafa-x) [inline src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md]: done

File: src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md (hunk 3 above)
Comment (mustafa-x) [inline src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md]: done

File: src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md (hunk 15 above)
Comment (mustafa-x) [inline src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md]: done

File: src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md (hunk 14 above)
Comment (mustafa-x) [inline src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md]: Done

File: src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md (hunk 1 above)
Comment (mustafa-x) [inline src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md]: done

File: src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md (hunk 12 above)
Comment (mustafa-x) [inline src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md]: done

File: src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md (hunk 10 above)
Comment (mustafa-x) [inline src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md]: done

**Comment by agektmr** [comment]
`payment-ux` directory is still remaining. You should remove them.

**Comment by mustafa-x** [comment]
Hey

I tried to remove it but some Gulp errors kept coming up, wanted to wait for @petele to take a look at it first. Everytime I removed and ran `gulp test` it showed a lot of errors to do with not finding the said deleted files.

**Pull request merged**